### DISCUSSION TITLE
Event Handling

### DISCUSSION THREAD

**Discussion by zharfanzahisham** [highfestiva/finplot#262] [General]
Hi, I'm curious as to whether there will be event handling features? I need to handle candlestick onclick event. Thank you in advance!

**Comment by highfestiva** [comment]
Hi, there's always the mysteriously named `finplot.set_time_inspector()` which takes a callback. You can use it for click, double-click or hover. See S&P 500 example for how to use it.

**Reply by zharfanzahisham** [reply]
Alright I'll check it out. Thank you blessed soul!
